res/xml/preferences.xml

```diff
@@ -27,8 +27,7 @@
 	        android:title="@string/prefs_select_oc_account"
 	        android:summary="@string/prefs_summary_select_oc_account"
 	        / -->
-	    <com.owncloud.android.ui.CheckBoxPreferenceWithLongTitle android:title="@string/prefs_pincode" android:key="set_pincode" 
-                        android:summary="@string/prefs_pincode_summary"/>
+	    <android.preference.CheckBoxPreference android:title="@string/prefs_pincode" android:key="set_pincode" />
```

Comment:
There’s really nothing needed to add. We shouldn’t add additional information just for the sake of it.